src/System.Xml.XmlSerializer/ref/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
+      uapaot;
```

Comment:
We should not ever have uapaot specific version of any ref assembly. The ref assemblies has to be same between uap and uapaot.